internal/proto/controller/servers/services/v1/credential.proto

```diff
@@ -35,3 +38,9 @@ message SshPrivateKey {
   // The optional passphrase of the private_key
   string private_key_passphrase = 30; // @gotags: `class:"secret"`
 }
+
+// Kv is a credential containing a set of key value pairs.
+message Kv {
```

Comment:
This can be dropped since we do not need to ever send KV credential to the worker